TelepresenceBot/server-unit/test/serverTest.js

```diff
@@ -0,0 +1,28 @@
+var should = require('should');
+var io = require('socket.io-client');
+var test = require('unit.js');
+
+var socketURL = 'http://0.0.0.0:5000'
+
+var options ={
+    transports: ['websocket'],
+    'force new connection': true
+};
+
+describe("TelepresenceBot Server: DummyTest ",function() {
+
+    it('Should add client to list of clients on connection.', function(done) {
```

Comment:
Ah yes, I just wanted to set up and initial structure but I should have tested this dummy code better 😆 5e994ec